src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int leg;
+    private int base;
+
+    public IsoscelesTrapezoid(String color, String name) {
+        setColor(color);
+        setName(name);
+    }
+
+    public int getBase() {
+        return base;
+    }
+
+    public void setBase(int base) {
+        this.base = base;
+    }
+
+    public int getLeg() {
+        return leg;
+    }
+
+    public void setLeg(int leg) {
+        this.leg = leg;
+    }
+
+    @Override
+    public double getArea() {
+        return super.getArea();
+    }
+
+    @Override
+    public void draw() {
+        super.draw();

```

Comment:
The draw method should be implemented to provide specific drawing logic for the IsoscelesTrapezoid. It should not just call the superclass method.